rise-google-sheet.html

```diff
@@ -292,17 +292,15 @@
         this._startTimer();
       },
 
-      _handleNoNetwork: function( resp ) {
+      _handleErrorsWithCachedData: function( resp ) {
```

Comment:
@stulees I think the name with cached data allow us to understand straight away what the method is doing. I like to be more descriptive on functions name so there is no need for comments. I can change it if you disagree. cheers